app/code/Magento/Customer/Test/Mftf/ActionGroup/AssertStoreFrontPasswordAutocompleteOffActionGroup.xml

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
- /**
-  * Copyright © Magento, Inc. All rights reserved.
-  * See COPYING.txt for license details.
-  */
--->
-<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="AssertStorefrontPasswordAutoCompleteOffActionGroup">
```

Comment:
Please, take a look at `app/code/Magento/Customer/Test/Mftf/Test/PasswordAutocompleteOffTest.xml:53`. This action group is used within the scope of the mentioned test. Why it's considered as redundant?

Reply:
After reading the PR description you should have noticed that I'm removing duplicated files with different names. The one I'm removing contains `StoreFront`, the one used in `PasswordAutocompleteOffTest` is `Storefront`

```xml
        <actionGroup ref="AssertStorefrontPasswordAutoCompleteOffActionGroup" stepKey="assertStorefrontPasswordAutoCompleteOff"/>
```

Reply:
Same thing as I described in comment to another action group.